docs/changelog.md

```
@@ -17,6 +19,8 @@
 
 
 ### Fixed
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)